src/scss/_main.scss

```diff
@@ -1313,6 +1345,34 @@ nav {
   }
 }
 
+.table-label {
+  font-size: 1.25rem;
+  color: var(--color-primary);
+  padding-top: 0.6rem 0 0 1rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.table-label .add-secret {
+  margin-right: 1em;
+}
+
+.field-description {
+  font-size: 1rem;
+  font-weight: 300;
+  margin-left: 1rem;
+}
+
+.field-header {
+  margin-top: 1rem;
+  margin-bottom: 0.25rem;
+}
+
+.-code {
```

Comment:
as a modifier, this should have a element it's attached to. I don't actually see this being used. If it was, would wrapping the content in `<code>` be sufficient?

Reply:
agreed! ill update

Reply:
oh that must have slipped in while dev'ing, its not even used. good catch, thank you!